pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -61,6 +62,8 @@
     private static final LongAdder bytesReceived = new LongAdder();
     private static final DecimalFormat dec = new DecimalFormat("0.000");
 
+    private static Recorder recorder = new Recorder(TimeUnit.DAYS.toMillis(90), 5);
```

Comment:
90 days converted into millis make a very high max. I am wondering whether that might impact the precision of the histogram itself (I think it works on buckets internally)

Reply:
yeah, it used internal key generated from value, so what about 30 days or other?